spanish-translations.md

```diff
@@ -0,0 +1,19 @@
+# Automatic Spanish Translations
+
+We're happy to announce that from now on we'll deliver Spanish translations to our
+tutorials (script and subtitles). How we made this possible? You may think that
+we hired a professional who takes care of translating the content, well, let me
+proudly say that you're wrong. As programmers, we love to automate processes, so, that's
+exactly what we did. We leverage the powerful translations service of *DeepL*. DeepL is
+a translating cloud service based on an AI algorithm which is surprisingly good. But, 
+this time it wasn't as easy as making an API call, we had to heavily customize our 
+content so it supports our custom MarkDown syntax and keeping the translations as 
+accurate as possible.
+
+Since this is a process run by machines, it's likely that you'll notice "details"
+in our translations. So, that's why we also delivered a way where you can easily submit
+a fix by sending a pull request. You only need to click on the "Edit on GitHub" button 
+which is at the right top corner of the script area, edit the script (or caption) file
+and commit the changes.
```

Comment:
I didn't realize it was that simple! Would we, at the moment, manually "transfer" those changes from the `.md` file to the `.vtt` file?

Reply:
If the user didn't modify both files, then, yes, we would have to apply manually the changes to the `.vtt` file